assets/src/apps/authoring/components/AdaptivityEditor/ActionMutateEditor.tsx

```diff
@@ -173,6 +172,7 @@ const ActionMutateEditor: React.FC<ActionMutateEditorProps> = (props) => {
           id={`action-mutate-value-${uuid}`}
           value={value}
           onChange={(e) => handleValueChange(e)}
+          onBlur={(e) => setIsDirty(true)}
```

Comment:
```suggestion
          onBlur={() => setIsDirty(true)}
```

There an eslint unused var warning on `e`